src/sci/impl/protocols.cljc

```diff
@@ -16,8 +16,7 @@
   #?(:clj (and (or (= 'Object sym)
                    (= 'java.lang.Object type))
                (= Object (interop/resolve-class ctx 'Object)))
-     :cljs (or (= 'object sym)
-               (= 'default sym))))
+     :cljs (= ::extend-default sym)))
```

Comment:
even in the impl before this PR, this would AFAICT never happen because it wasn't possible to extend a protocol to either `'default` or `'object`